test/scripts/base-host-check.sh

```diff
@@ -98,6 +98,33 @@ check_ca_cert() {
     fi
 }
 
+check_modularity() {
+    echo "📗 Checking enabled modules"
+
+    # Verify modules that are enabled on a system, if any. Modules can either be enabled separately
+    # or they can be installed through packages directly. We test both cases here.
+    #
+    # Caveat is that when a module is enabled yet _no_ packages are installed from it this breaks.
+    # Let's not do that in the test?
+
+    modules_expected_0=$(jq -rc '.blueprint.enabled_modules[]? | .name + ":" + .stream' "${config}")
+    modules_expected_1=$(jq -rc '.blueprint.packages[]? | select(.name | startswith("@") and contains(":")) | .name' "${config}" | cut -c2-)
+
+    modules_expected="${modules_expected_0}\n${modules_expected_1}"
```

Comment:
I think this isn't working exactly as intended.  I'm seeing the following in the test logs:
- `📗 Module expected: nodejs:18\n` in the [enabled modules](https://gitlab.com/redhat/services/products/image-builder/ci/images/-/jobs/9335396048) test.
- `📗 Module expected: \nnodejs:18` in the [modularity in packages](https://gitlab.com/redhat/services/products/image-builder/ci/images/-/jobs/9335396051) test.

It also makes me wonder if the test will succeed when we have more than one module defined in the packages, or in the enabled_modules, or when we have combinations.

I suppose it's fine for now, the actual feature works, so I wont block, but let's check and fix up the test script soon.